contracts/0.8.25/vaults/Dashboard.sol

```diff
@@ -210,44 +209,49 @@ contract Dashboard is AccessControlEnumerable {
      * @notice Transfers ownership of the staking vault to a new owner.
      * @param _newOwner Address of the new owner.
      */
-    function transferStVaultOwnership(address _newOwner) external virtual onlyRole(DEFAULT_ADMIN_ROLE) {
-        _transferStVaultOwnership(_newOwner);
+    function transferStakingVaultOwnership(address _newOwner) external {
+        _transferStakingVaultOwnership(_newOwner);
     }
 
     /**
      * @notice Disconnects the staking vault from the vault hub.
```

Comment:
Need to explain multi-permission nature of this method. Mean, you may need three different roles to use it.